app/src/main/java/kth/jjve/xfran/models/Event.java

```diff
@@ -0,0 +1,56 @@
+package kth.jjve.xfran.models;
+/*
+Function: defines event
+Activity: CalendarViewActivity
+Jitse van Esch, Elisa Perini & Mariah Sabioni
+based on: https://github.com/codeWithCal/CalendarTutorialAndroidStudio/tree/WeeklyCalendar
+ */
+
+import java.time.LocalDate;
+import java.time.LocalTime;
+import java.util.ArrayList;
+
+public class Event {
+
+    public static ArrayList<Event> eventsList = new ArrayList<>();
+
+    private String name;
+    private LocalDate date;
+    private LocalTime startTime;
+    private LocalTime endTime;
+
+    public Event(String name, LocalDate date, LocalTime startTime, LocalTime endTime) {
+        this.name = name;
+        this.date = date;
+        this.startTime = startTime;
+        this.endTime = endTime;
```

Comment:
I would allow saving an event with no start/end time. App crashes if I try that now. I think it makes sense that a workout can be planned anytime of a given day.

Reply:
There's actually a reason for that, it will be easier to implemet the google calendar feature with start and end time first, without time (basically a daily event), seems to be a tad more complicated, that's why I wanted to have the event with start and end times working first